org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/CompoundModel.java

```diff
@@ -226,6 +226,7 @@ public int hashCode() {
 		result = prime * result + (int)(durationBits ^ (durationBits >>> 32));
 		return result;
 	}
+
 	@Override
 	public boolean equals(Object obj) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)